### DISCUSSION TITLE
No sound/audio coming from speakers

### DISCUSSION THREAD

**Discussion by niteguide** [music-assistant/support#2627] [Q&A] [answered]
Generally speaking MA works good and I am very grateful for the rapid development of the service. A big thank you to all the people involved! 

There are however some points I can't get completely under control. I have a HomePod Mini and a Google Home Mini grouped under one Universal Player. On some occasions suddenly the audio stops. When I look into MA everything appears to be normal and the song seems to play. 
I then check the entity status of HomePod added via AppleTV integration and Google Home Mini added via Chromcast and one went to idle the other to buffering. 
I then tried to manually skip a track inside MA and the sound is back instantly.
Compression is deactivated for HomePod, Chromecast does not have such an option, I think. 

Any ideas what is going on here and how to tweak the setup to not have this problem?

**Comment by OzGav** [comment]
Universal groups work on a best effort basis so unusual behaviour may occur. Having said that when resources allow this will be looked at further.

**Reply by niteguide** [reply]
Thank you! I was already vigorously scanning through forums in search for a solution. Maybe you could include a "attention-notice" within Universal Player settings concerning what might not work.
I of course hope that you soon have time to give the Universal Player some attention regarding that matter!

**Reply by OzGav** [reply, marked as answer]
This is in the docs 

> UGP is on best-effort basis and might not work correctly with all player models or give unexpected results. For best results use standalone or syncgroup playback
